src/index.html

```diff
@@ -10,13 +10,54 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"class="logo">
```

Comment:
There should not be a blank line between the `header` and `a` tags since they are parent and child elements.